add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: There is no error handling for invalid or missing input data.

Recommendations:
- Implement input validation and error handling to ensure data integrity. Use PHP's filter functions or a validation library.
- Example:
  ```php
  if (empty($title) || empty($author) || !filter_var($year, FILTER_VALIDATE_INT)) {
      echo "Invalid input data.";
      return;
  }
  ```